src/engraving/libmscore/chord.cpp

```diff
@@ -1391,6 +1391,9 @@ void Chord::processSiblings(std::function<void(EngravingItem*)> func) const
     }
     for (Note* note : _notes) {
         func(note);
+        for (NoteDot* dot : note->dots()) {
+            func(dot);
```

Comment:
It might seem Note itself should do this, but in principle a Note can be selected independently of its augmentation dots, whereas selecting a *chord* always means selecting all the sub-components.